docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Macro Behavior
+sidebar_label: Macros
+---
+
+## Summary
+
+The macro behavior allows configuring a list of other behaviors to invoke
+when the macro is pressed, including support for completing part of the binding list
+once the macro is released.
+
+## Macro Definition
+
+Each macro you want to use in your keymap gets defined first, then bound in your keymap.
+
+A macro definition looks like:
+
+```
+/ {
+    macros {
+        zed_em_kay: zmk_macro {
+            label = "ZMK_MAC";
```

Comment:
Now that we have the C macro, we could make this example more consistent. 

Also, I personally like things repeating throughout the binding, name, and label. As a user, it's helpful for a copy/paste find/replace workflow.

```suggestion
        zed_em_kay: zed_em_kay {
            label = "ZMK_zed_em_kay";
```

Reply:
Yup. Good call. Fixed.